R/getCPR.R

```diff
@@ -1,44 +1,42 @@
 #' Get cells per region
-#' 
-#' Counts how many cells each region has and returns it as vector
-#' 
-#' 
+#'
+#' Counts how often each element of the provided subdimension exists in the given
+#' data set. Originally created to count the number of cells in a region (this
+#' is also where its name originates from) it can now be used to count elements
+#' of any subdimension via the dim argument.
```

Comment:
How about renaming the function and adding an alias/redirection from getCPR to the new name for backwards compatibility? To me `getCPR` is already confusing because of the abbreviation and if the function now even does a lot more I think a better name could really make code easier to understand.